<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterDialog</class>
 <widget class="QDialog" name="UpdaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Citron Updater</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/default/citron.ico</normaloff>:/icons/default/citron.ico</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>12</number>
   </property>
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Checking for updates...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Please wait while we check for available updates...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="updateInfoGroup">
     <property name="title">
      <string>Update Information</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="currentVersionLabel">
        <property name="text">
         <string>Current Version:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentVersionValue">
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="latestVersionLabel">
        <property name="text">
         <string>Latest Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="latestVersionValue">
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="releaseDateLabel">
        <property name="text">
         <string>Release Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="releaseDateValue">
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="changelogGroup">
     <property name="title">
      <string>Changelog</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="changelogLayout">
      <item>
       <widget class="QTextBrowser" name="changelogText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;No changelog available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroup">
     <property name="title">
      <string>Progress</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="progressLayout">
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="text">
         <string>Preparing...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="downloadInfoLabel">
        <property name="text">
         <string></string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <!-- Start of added widgets -->
   <item>
    <layout class="QHBoxLayout" name="appImageSelectorLayout">
     <item>
      <widget class="QLabel" name="appImageSelectorLabel">
       <property name="text">
        <string>Select AppImage:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="appImageSelector"/>
     </item>
    </layout>
   </item>
   <!-- End of added widgets -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download Update</string>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="text">
        <string>Restart Citron</string>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
